test/src/common/harnessWal.h

```diff
@@ -0,0 +1,38 @@
+#ifndef TEST_COMMON_HARNESS_WAL_H
+#define TEST_COMMON_HARNESS_WAL_H
+#include "common/type/buffer.h"
+#include "common/type/param.h"
+#include "common/walFilter/postgresCommon.h"
+
+typedef enum InsertRecordFlags
+{
+    NO_FLAGS = 0,
+    INCOMPLETE_RECORD = 1 << 0,
+    OVERWRITE = 1 << 1,
+    COND_FLAG = 1 << 2,
+    NO_COND_FLAG = 1 << 3,
+    ZERO_REM_LEN = 1 << 4,
+    WRONG_REM_LEN = 1 << 5
+}InsertRecordFlags;
+
+typedef struct InsertXRecordParam
+{
+    VAR_PARAM_HEADER;
+    uint16_t magic;
+    uint32_t begin_offset;
```

Comment:
There is a mixture of styles in fields naming - both snake_case and camelCase.  Lets make it consistent.